contracts/Holograph.sol

```diff
@@ -122,142 +130,100 @@ contract Holograph is Admin, Initializable, IHolograph {
       address treasury
     ) = abi.decode(data, (uint32, address, address, address, address, address, address));
     assembly {
-      sstore(0x5705f5753aa4f617eef2cae1dada3d3355e9387b04d19191f09b545e684ca50d, origin())
-
-      sstore(0xf659863303d91045cf6f5789ae687c591017a1efd53ebb2eec518fb10873ecb8, chainType)
-
-      sstore(0x03be85923973d3197c19b1ad1f9b28c331dd9229cd80cbf84926b2286fc4563f, bridge)
-      sstore(0x7eefc8e705e14d34b5d1d6c3ea7f4e20cecb5956b182bac952a455d9372b87e2, factory)
-      sstore(0x23e584d4fb363739321c1e56c9bcdc29517a4c57065f8502226c995fd15b2472, interfaces)
-      sstore(0x7bef7d8d97f57f9aa64de319c8598b5cdc7c3d2715fc02428415a98281ca6bdc, operator)
-      sstore(0x460c4059d72b144253e5fc4e2aacbae2bcd6362c67862cd58ecbab0e7b10c349, registry)
-      sstore(0xc8f5846d0f0d68cef76d4d10d7d189845e41b44d1dbdc208ed0f8961f993af5f, treasury)
+      sstore(_adminSlot, origin())
+      sstore(_chainTypeSlot, chainType)
+      sstore(_bridgeSlot, bridge)
+      sstore(_factorySlot, factory)
+      sstore(_interfacesSlot, interfaces)
+      sstore(_operatorSlot, operator)
+      sstore(_registrySlot, registry)
+      sstore(_treasurySlot, treasury)
     }
     _setInitialized();
     return IInitializable.init.selector;
   }
 
-  /**
-   * @dev Returns an integer value of the chain type that the factory is currently on.
-   * @dev For example:
-   *                   1 = Ethereum mainnet
-   *                   2 = Binance Smart Chain mainnet
-   *                   3 = Avalanche mainnet
-   *                   4 = Polygon mainnet
-   *                   etc.
-   */
-  function getChainType() public view returns (uint32 chainType) {
-    // The slot hash has been precomputed for gas optimizaion
-    // bytes32 slot = bytes32(uint256(keccak256('eip1967.Holograph.Bridge.chainType')) - 1);
```

Comment:
Do you think it's not important to document what the slots were generated from in the comments still? Maybe it's worth putting one comment at the top of https://github.com/holographxyz/holograph-protocol/pull/36/files#diff-d7e82dd160b5facd64a9d67736cacc298beb65cc58c3ae9e7161f7ac9f686181R111 that explains the formula. Something like: 

`bytes32 slot = bytes32(uint256(keccak256('eip1967.Holograph.<contract-name>.<property-name>')) - 1);`

Reply:
Sure, we can add it as commentary on top where storage slots are computed and stored as constants.